### ISSUE TITLE
[Feature] Offset pagination

### ISSUE THREAD

**Issue by 0xOlias** [ponder-sh/ponder#901]
The `findMany` store method and the GraphQL API support cursor pagination as specified [here](https://ponder.sh/docs/query/graphql#pagination).

A number of users have also requested offset pagination to support a "jump to page" feature, eg `/?page=1`. There are performance considerations here, but now that we have custom database indexes, perf issues can often be mitigated in userland.

**Comment by hskang9** [comment]
I agree with this idea. Offset is inefficient, but it is effective in making pagination UI components. How can I contribute to this?
